File: client/src/app/shared/models/depotPrintTrack.ts
export interface IDepotPrintTrack {
    id: number;
    investmentInitId: number;
    paymentRefNo: string;
    paymentDate: Date;
    depotId: string;
    depotName: string;
    remarks: string;
    employeeId: number;
    lastPrintTime : Date;
    printCount : number;
    chequeNo: string;
    bankName: string;
}

export class DepotPrintTrack implements IDepotPrintTrack {
    id: number = 0;
    investmentInitId: number;
    paymentRefNo: string;
    paymentDate: Date;
    depotId: string;
    depotName: string;
    remarks: string;
    employeeId: number;
    lastPrintTime : Date = new Date();
    printCount : number;
    chequeNo: string;
    bankName: string;
}

File: Core/Entities/DepotPrintTrack.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace Core.Entities
{
    public class DepotPrintTrack : BaseEntity
    {
        public int? InvestmentInitId { get; set; }
        [ForeignKey("InvestmentInitId")]
        public InvestmentInit InvestmentInit { get; set; }
        public string PayRefNo { get; set; }
        public string SAPRefNo { get; set; }
        public string DepotId { get; set; }
        public string DepotName { get; set; }
        public string Remarks { get; set; }
        public string PaymentRefNo { get; set; }
        public DateTimeOffset? PaymentDate { get; set; }
        public int? EmployeeId { get; set; }
        public Employee Employee { get; set; }
        public DateTimeOffset? LastPrintTime { get; set; }
        public int PrintCount { get; set; }  
        public string ChequeNo { get; set; }
        public string BankName { get; set; }
    }
}